server/constants/participant-fields.ts

```diff
@@ -0,0 +1,21 @@
+export const participantFields = [
+  'id',
+  'firstName',
+  'lastName',
+  'postalCodeFsa',
+  'preferredLocation',
+  'program',
+  'educationalRequirements',
+  'driverLicense',
+  'indigenous',
+  'nonHCAP',
+  'interested',
+  'crcClear',
+  'callbackStatus',
+  'statusInfo',
+  'userUpdatedAt',
+  'distance',
+  'progressStats',
+  'postHireStatuses',
+  'rosStatuses',
+];
```

Comment:
@fw-noel  theres a few other fields/ conditional questions from the docs, should these be here? 

, I also have: `experienceWithMentalHealthOrSubstanceUse`, `currentOrMostRecentIndustry`, `roleInvolvesMentalHealthOrSubstanceUse`, and `otherIndustry`, I don't believe they're 'required' though

Reply:
I'll add them.